src/includes/getting-started-config/node.mdx

```diff
@@ -4,7 +4,11 @@ Once this is done, Sentry's Node SDK captures all transactions and unhandled exc
 import * as Sentry from "@sentry/node";
 
 // Importing @sentry/tracing patches the global hub for tracing to work.
-import * as Tracing from "@sentry/tracing";
+import "@sentry/tracing";
+
+// If you want to use `Tracing` object in your project, you can use named imports such as:
+// import * as Tracing from "@sentry/tracing"
```

Comment:
Nit:

```suggestion
// If you want to use `@sentry/tracing` in your project directly, use a named import instead:
// import * as SentryTracing from "@sentry/tracing"
```

(If you do this, I'd suggest switching the CJS to `SentryTracing` as well.)

Reply:
Done 👍